docs/installation/server-config/index.rst

```diff
@@ -83,16 +77,18 @@ Here are some common configuration properties that you might want to customize.
 .. _ksql-auto-offset-reset:
 
 ^^^^^^^^^^^^^^^^^
-auto.offset.reset
+ksql.streams.auto.offset.reset
 ^^^^^^^^^^^^^^^^^
 
-Determines what to do when there is no initial offset in Kafka or if the current offset does not exist on the server. The
-default value in KSQL is ``latest``, which means all Kafka topics are read from the latest available offset. For example,
-to change it to earliest by using the KSQL command line:
+Determines what to do when a KSQL query has not committed any offsets and has thus not marked progress on its 
+input Kafka topic(s). The default value in KSQL is ``latest``, which means KSQL queries will read Kafka topics 
```

Comment:
you can just use `topics` rather than `topic(s)` - the plural form is inclusive of singular.